.github/workflows/pull_requests.yaml

```diff
@@ -9,93 +9,93 @@ on:
-        run: ./test/run.sh --acceptance-only
+  # Run-Swagger:
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/checkout@v3
+  #     - name: Set up Go
+  #       uses: actions/setup-go@v3
+  #       with:
+  #         go-version: 1.19
+  #         cache: true
+  #     - name: Run Swagger
+  #       run: ./tools/gen-code-from-swagger.sh
+  #     - name: Error on change
+  #       run: |
+  #         # check if anything is different
+  #         CHANGED=$(git status -s | wc -l)
+  #         if [ "$CHANGED" -gt 0 ]; then
+  #           echo "Please run ./tools/gen-code-from-swagger.sh script and commit changes"
+  #           exit 1
+  #         else
+  #           exit 0
+  #         fi
+  # Unit-Tests:
+  #   needs: [Run-Swagger]
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/checkout@v3
+  #     - name: Set up Go
+  #       uses: actions/setup-go@v3
+  #       with:
+  #         go-version: 1.19
+  #         cache: true
+  #     - name: Unit test
+  #       run: ./test/run.sh --unit-only
+  #     - name: Codecov Unit
+  #       if: ${{ (github.ref_type == 'branch') && (github.ref_name != 'master') }}
+  #       uses: codecov/codecov-action@v3
+  #       with:
+  #         fail_ci_if_error: true
+  #         files: ./coverage-unit.txt
+  #         flags: unittests
+  #         verbose: true
+  # Integration-Tests:
+  #   needs: [Run-Swagger]
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/checkout@v3
+  #     - name: Set up Go
+  #       uses: actions/setup-go@v3
+  #       with:
+  #         go-version: 1.19
+  #         cache: true
+  #     - name: Integration test
+  #       run: ./test/run.sh --integration-only
+  #     - name: Codecov Integration
+  #       if: ${{ (github.ref_type == 'branch') && (github.ref_name != 'master') }}
+  #       uses: codecov/codecov-action@v3
+  #       with:
+  #         fail_ci_if_error: true
+  #         files: ./coverage-integration.txt
+  #         flags: integration
+  #         verbose: true
+  # Modules-Acceptance-Tests:
+  #   needs: [Run-Swagger]
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/checkout@v3
+  #     - name: Set up Go
+  #       uses: actions/setup-go@v3
+  #       with:
+  #         go-version: 1.19
+  #         cache: true
+  #     - name: Acceptance tests (modules)
+  #       run: ./test/run.sh --acceptance-module-tests-only
+  # Acceptance-Tests:
+  #   needs: [Run-Swagger]
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/checkout@v3
+  #     - name: Set up Go
+  #       uses: actions/setup-go@v3
+  #       with:
+  #         go-version: 1.19
+  #         cache: true
+  #     - name: Acceptance tests (modules)
+  #       run: ./test/run.sh --acceptance-only
   Push-Docker:
-    needs: [Acceptance-Tests, Modules-Acceptance-Tests, Unit-Tests, Integration-Tests]
+    # needs: [Acceptance-Tests, Modules-Acceptance-Tests, Unit-Tests, Integration-Tests]
```

Comment:
remove comments? :)

Reply:
Haha, sorry, I put the status to "Draft" again as I started implementing Andrzej's review feedback. will fix this before marking it as ready again.